schematic/models/validate_attribute.py

```diff
@@ -1617,6 +1770,10 @@ def _run_validation_across_target_manifests(
         # Start timer
         start_time = perf_counter()
 
+        # if not target_manifest_ids:
+        # target_manifet_present = False
+        # return (start_time, target_manifet_present)
+        # else:
```

Comment:
Same as above

Reply:
Thanks, this was a previous approach that didnt work out and I forgot to remove it completely.